projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/style.scss

```diff
@@ -0,0 +1,51 @@
+.value-section {
+    display: flex;
+    justify-content: space-between;
+
+    &__heading {
+        font-size: 12px;
+        color: var(--jp-gray-100);
+        font-weight: 500;
+    }
+
+    &__last-scan {
+        display: flex;
+        align-items: center;
+        margin-top: var(--spacing-base);
+        column-gap: 1px;
+        position: absolute;
+        right: calc(var(--spacing-base)* 3);
+        span {
+            font-size: var(--font-body-extra-small);
+            line-height: var(--font-title-small);
+            color: var(--jp-gray-50);
+        }
+    }
+
+    &__tooltip-button {
+        display: flex;
+        align-items: center;
+        background: transparent;
+        border: none;
+        color: #888888;
```

Comment:
It seems like this is a different grey than the one being used on the text next to it. Is there any reason we can't use the same grey for both?

Reply:
Good point!  Hmm, not sure where I got that color from. 🤔
Yes, we should make them the same. 👍